### PR TITLE
Add compliance takeover

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build FIPS certification and CIS compliance takeover
- Add to `templates/takeovers/index.html`
- Add to `templates/index.html`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001
- Check takeover is visible on homepage and matches description on [issue](https://github.com/canonical-web-and-design/web-squad/issues/3209)
- Check CTA 

## Issue / Card

Fixes [#3209](https://github.com/canonical-web-and-design/web-squad/issues/3209)

## Screenshots

image

### CODE REVIEW

**Comment by Khutch25** [comment]
Looks great! Could the compliance illustration be a white version instead of a gray?

**Comment by bethcollins92** [comment]
@Khutch25 Changed the compliance illustration to white - take a look

File: templates/takeovers/_fips_certification_cis_compliance.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% with title="FIPS certification and CIS compliance with Ubuntu",
+subtitle="Learn about Ubuntu CIS and FIPS certified components to enable operating environments under compliance regimes like FedRAMP, HIPAA, PCI and ISO.",
+header_image="https://assets.ubuntu.com/v1/b576398c-compliance-wht.svg",
+image_width="400",
+image_height="234",
+image_hide_small="true",
+equal_cols="true",
+primary_url="https://www.brighttalk.com/webcast/6793/432536?utm_source=Takeover",
+primary_cta="Register",
```
Comment (pmahnke) [inline templates/takeovers/_fips_certification_cis_compliance.html:9]:
```suggestion
primary_cta="Register for the webinar",
```

Sorry, I thought Kelly was changing it.

**Review by pmahnke — APPROVED** [review]
+1 with the CTA change

**Comment by Khutch25** [comment]
Looks great! Thank you so much @bethcollins92 and @pmahnke!!

**Comment by Khutch25** [comment]
Opening again to see if the word "environments" can be removed from the takeover please! @bethcollins92

**Pull request merged**